client/packages/editor/src/components/tabs/CodeTab.tsx

```diff
@@ -2,18 +2,28 @@ import { useData } from '../../context';
 import { TabProps, TabState } from '../props';
 import LabelInput from '../widgets/label/LabelInput';
 import { CodeEditor } from '../widgets';
+import Checkbox from '../widgets/checkbox/Checkbox';
+
+const SECURITY_SYSTEM = 'system';
```

Comment:
I have a task for that: hopefully the client soon can forget this special value :relaxed: 

i think we will introduce:
`checkPermissions = true/false`
or if someone has a preference:
`asSystem = true/false`

Reply:
both are good for me. 
Maybe the one which is more the truth? I mean does this flag mean that there are no permission checks (but the same user or is the script step run with the system user?